### PR TITLE
week 2 homework

### PR DESCRIPTION
**Pull request by shanhaider23**

### CODE REVIEW

File: week2/task3/ex3.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        Hours += hours;
+        if (Hours > 23)
+        {
+            Hours -= 24;
+        }
+        if (Hours < 0)
+        {
+            Hours = Hours + 24;
+        }
```
Comment (LaurentiuCalin) [inline week2/task3/ex3.cs:41]: A day on Jupiter lasts 10h therefore the AddHours method should reflect that time

File: week2/task4/ex4.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+time.AddMinutes(-20);
+PrintTime(time);
+
+
+void PrintTime(JupiterTime time)
+{
+
+    Console.WriteLine($"{time.Hours}:{time.Minutes}");
+
+
```
Comment (LaurentiuCalin) [inline week2/task4/ex4.cs:13]: Try to avoid using new lines if they are not used to splitting functionality or logic

File: week2/task6/ex6.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+{
+    List<JupiterTime> Times = new List<JupiterTime>();
+    public void AddTime(JupiterTime time)
+    {
+        Times.Add(time);
+    }
+
+    public void Inform()
+    {
+        if (Times == null) Console.WriteLine("No timers added yet.");
```
Comment (LaurentiuCalin) [inline week2/task6/ex6.cs:19]: Times will never be null as you are instantiating it every time by doing `List<JupiterTime> Times = new List<JupiterTime>();`

A better check here would be for the .Count of the Times

**Review by LaurentiuCalin — COMMENTED** [review]
Hello 😊  
Great job overall! 
I have left a few comments, let me know if you have any questions

**Comment by shanhaider23** [comment]
Thanks for review I have updated it

**Pull request merged**